acceptance-tests/src/test-support/java/tech/pegasys/artemis/test/acceptance/dsl/ArtemisNode.java

```diff
@@ -92,28 +93,18 @@ private BeaconHead getCurrentBeaconHead() throws IOException {
   }
 
   private URI getRestApiUrl() {
-    return URI.create("http://127.0.0.1:" + config.getRestApiPortNumber());
+    return URI.create("http://127.0.0.1:" + container.getMappedPort(config.getRestApiPortNumber()));
   }
 
   public void stop() {
     if (!started) {
       return;
     }
     LOG.debug("Shutting down");
-    try {
-      process.destroy();
-      if (!process.waitFor(1, TimeUnit.MINUTES)) {
-        LOG.warn("Didn't shutdown in a timely fashion, being more aggressive");
-        process.destroyForcibly();
-      }
-
-      if (!configFile.delete() && configFile.exists()) {
-        throw new IOException("Failed to delete config file: " + configFile);
-      }
-      deleteRecursively(workDir);
-    } catch (final IOException | InterruptedException e) {
-      throw new RuntimeException(e);
+    if (!configFile.delete() && configFile.exists()) {
```

Comment:
Do we need to explicitly delete this tmp file?

Reply:
Technically we could leave it to the JVM to delete on exit but especially in tests I tend to explicitly delete temp files when I'm done with them to minimise the number of files that get left around if the JVM happens to crash.